src/plugins/kibana/public/settings/sections/indices/_create.js

```diff
@@ -23,7 +23,7 @@ uiModules.get('apps/settings')
 
   // this and child scopes will write pattern vars here
   const index = $scope.index = {
-    name: 'logstash-*',
+    name: config.get('kibana.indexPatternPlaceholder', 'logstash-*'),
```

Comment:
This config service is actually not the same one that's on the server. Is there a reason to put this in the kibana.yml vs the advanced config settings?